src/main.rs

```diff
@@ -0,0 +1,36 @@
+use axum::{
+  routing::{get, post},
+  Router,
+};
+use reqwest::Client;
+
+mod shutdown_signal;
+use shutdown_signal::shutdown_signal;
+
+mod speech;
+use speech::speech;
+
+#[tokio::main]
+async fn main() {
+  tracing_subscriber::fmt::init();
+
+  let client = Client::new();
```

Comment:
Do we need to set a `timeout` for `client`? It appears that neither Reqwest nor Axum sets a timeout on clients.

Reply:
Good point, I'll take a look after.